packages/chart/src/CdcChartComponent.tsx

```diff
@@ -593,25 +542,9 @@ const CdcChart = ({
       setSeriesHighlight(newConfig.legend?.seriesHighlight)
     }
 
-    setConfig(newConfig)
-  }
+    dispatch({ type: 'SET_CONFIG', payload: newConfig })
 
-  // Sorts data series for horizontal bar charts
```

Comment:
Are you using a different sort fn here?

Reply:
we dont need to sort any bars. it was very old code config.sort  which we dont have logic. also sortings happens now automatically in useScales for time, and date types for vertical bars.Can you re confirm it ?